solrman/solrmanapi/cloudstatus.go

```diff
@@ -20,12 +20,14 @@ import "time"
 type SolrCloudStatus map[string]*SolrNodeStatus // keys are hostnames
 
 type SolrNodeStatus struct {
-	Hostname      string                     // the node's hostname, as determined by gethostbyaddr
-	NodeName      string                     // the node's identifier within SolrCloud (e.g. "1.1.1.1:8983_solr")
-	Cores         map[string]*SolrCoreStatus // keys are core names
-	Zone          string                     // cloud zone where host exists
-	DiskSize      float64
-	IndexDiskSize float64
+	Hostname               string                     // the node's hostname, as determined by gethostbyaddr
+	NodeName               string                     // the node's identifier within SolrCloud (e.g. "1.1.1.1:8983_solr")
+	Cores                  map[string]*SolrCoreStatus // keys are core names
+	Zone                   string                     // cloud zone where host exists
+	DiskSize               float64
+	IndexDiskSize          float64
+	IndexDiskSizeTotal     int64
+	IndexDiskSizeAvailable int64
```

Comment:
Just curious why we need `IndexDiskSizeAvailable` ?

Reply:
We need both total and available so we can tell what percentage full the disk is. We need `IndexDiskSizeAvailable` too so we can calculate if moving a shard onto the disk will cause it to go over the 90% threshold.

Reply:
i thought we get raw `DiskSize` from gcp-computeInsatnce-api`, and we just need used space.